docs/faq/troubleshooting/not-a-member-of-the-organization.md

```diff
@@ -4,9 +4,18 @@ When you see this message, it means that Codacy can't analyze a commit because t
 
 There may be different reasons for this to happen:
 
--   The user making the commit hasn't [signed in to Codacy and joined the organization](../../getting-started/getting-started-with-codacy.md) yet. Or, if the user doesn't belong to your organization, [add the user as an author](../../organizations/adding-and-managing-authors.md) instead.
+-   **The user making the commit hasn't [signed in to Codacy and joined the organization](../../getting-started/getting-started-with-codacy.md) yet.** Or, if the user doesn't belong to your organization, an organization owner must [add the user as an author](../../organizations/adding-and-managing-authors.md) instead.
 
--   The commit email address isn't associated with the account of a Codacy user. Codacy automatically associates the email addresses from the Git provider accounts to the Codacy accounts when users sign in to Codacy. Make sure that the user configures the missing email address on their Git provider account, and that the user logs in again on Codacy for the change to take effect.
+-   **The commit email address isn't associated with the account of a Codacy user.** Codacy automatically associates the email addresses from the Git provider accounts to the Codacy accounts when users sign in to Codacy. Make sure that the user configures the missing email address on their Git provider account, and that the user logs in again on Codacy for the change to take effect.
 
-!!! note
-    Please note that Codacy doesn't allow adding email addresses ending with `.local`. In this case, please use the email address that's associated with your Codacy account instead.
+    You can check which email address is associated with a commit by hovering the cursor on the name of the contributor on the page for the commit:
+
+    ![Checking the email address for a commit](images/not-a-member-of-the-organization-commit.png)
+
+    To verify which email addresses are associated with the Codacy account, the user must click on their avatar on the top right-hand corner, select **Your account**, and open the page **Emails**:
+
+    ![Email addresses associated with a user account](images/not-a-member-of-the-organization-account.png)
+
+
+    !!! note
+        Please note that Codacy doesn't allow adding email addresses ending with `.local`. In this case, please use the email address that's associated with your Codacy account instead.
```

Comment:
@IGFCoimbra, I need more information to improve this part. Could you describe the exact issue that customers face when dealing with email addresses ending with `.local`?